### ISSUE TITLE
DistributedFusedLAMB seems incorrect

### ISSUE THREAD

**Issue by Ruinhuang** [NVIDIA/apex#1041]
Hi, I try to use DistributedFusedLAMB optimizer for distributed training of resnet50 model, but the convergence of loss curve is very slow, so I follow the testing method of link below to test DistributedFusedLAMB:
https://github.com/kexinyu/apex/blob/kyu/dist_lamb_math/tests/L0/run_optimizers/test_dist_adam.py
the result shows weight difference between DistributedFusedLAMB and FusedLAMB is bigger than 1e-3 within training step 20, with training step increases, it becomes worse.
I am wondering whether it is right.
@kexinyu I see you did a lot of perfect work on DistributedFusedLAMB, could you help me please if you have bandwidth, thank you very much : )
